### PR TITLE
feat: alert block

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes # https://vsf.atlassian.net/jira/software/c/projects/SFUI2/boards/69?modal=detail&selectedIssue=SFUI2-654

# Scope of work

Alert block 

Blocks created as shown in examples in figma (not every alert has an icon as in task)

Buttons also added (without functionality) to adjust figma design (task was created before designs and it was assumed that there won't be buttons in the first phase but we already have them in designs) 

# Screenshots of visual changes
Screenshot 2023-04-12 at 08 16 50
Screenshot 2023-04-12 at 08 16 45
Screenshot 2023-04-12 at 08 16 42
Screenshot 2023-04-12 at 08 16 38
Screenshot 2023-04-12 at 08 16 34

Screenshot 2023-04-12 at 07 12 06

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/preview/next/pages/showcases/Alert/AlertPositive.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <div
+      role="alert"
+      className="flex items-start md:items-center bg-positive-100 pr-2 pl-4 ring-1 ring-positive-200 typography-text-sm md:typography-text-base py-1 rounded-md"
+    >
+      <SfIconCheckCircle className="mr-2 my-2 text-positive-700" />
+      <p className="py-2 mr-2">The product has been added to the cart.</p>
+      <button
+        type="button"
+        className="p-1.5 md:p-2 ml-auto rounded-md text-positive-700 hover:bg-positive-200 active:bg-positive-300 hover:text-positive-800 active:text-positive-900"
+        aria-label="Close alert"
```
Comment (aniamusial) [inline apps/preview/next/pages/showcases/Alert/AlertPositive.tsx:16]: suggestion: if we have all those alerts on the same page, then maybe we can differentiate the aria labels, so its not just "close alert" 4 times? 😄 fe. 'Close positive alert' etc.

File: apps/preview/next/pages/showcases/Alert/AlertError.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { SfIconClose } from '@storefront-ui/react';
+
+export default function AlertError() {
+  return (
+    <div
+      role="alert"
+      className="flex items-start md:items-center bg-negative-100 pr-2 pl-4 ring-1 ring-negative-300 typography-text-sm md:typography-text-base py-1 rounded-md"
+    >
+      <p className="py-2 mr-2">The password change was failed.</p>
+      <button
```
[the remark this answers is not in the document]
Comment (justyna-13) [reply apps/preview/next/pages/showcases/Alert/AlertError.tsx:12]: I thought that it'd be too much of overriding styles and even on figma they are called custom buttons

**Review by aniamusial — APPROVED** [review]

File: apps/docs/components/blocks/Alert.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+description: Alert is a notification that keeps people informed of the status of the system and which may or not require the user respond.
+
+---
+# Alert
+
+{{ $frontmatter.description }}
+
+## Alert neutral
+
+Simply version of alert that has neutral grey color.
```
[the remark this answers is not in the document]
Comment (justyna-13) [reply apps/docs/components/blocks/Alert.md]: https://github.com/vuestorefront/storefront-ui/pull/2590/commits/483678a16251c1a4e9ea97164ef1dc6a142aee78

File: apps/preview/next/pages/showcases/Alert/AlertPositive.tsx (hunk 1 above)
Comment (justyna-13) [reply apps/preview/next/pages/showcases/Alert/AlertPositive.tsx]: https://github.com/vuestorefront/storefront-ui/pull/2590/commits/483678a16251c1a4e9ea97164ef1dc6a142aee78

**Review by Szymon-dziewonski — APPROVED** [review]
Good job!

**Pull request merged**